tests/phpunit/tests/l10n/wpLocaleSwitcher.php

```diff
@@ -382,6 +388,79 @@ public function test_multiple_switches_to_site_locale_and_user_locale() {
 		$this->assertSame( 'This is a dummy plugin', $actual );
 	}
 
+	/**
+	 * @ticket 39210
+	 */
+	public function test_switch_reloads_plugin_translations_outside_wp_lang_dir() {
```

Comment:
Why the test is failing right now:

1. Including `custom-internationalized-plugin.php` will trigger the `load_plugin_textdomain()` call
2. This adds a cache entry to the registry for `custom-internationalized-plugin, en_US, /var/www/tests/phpunit/data/plugins/custom-internationalized-plugin/languages/`
3. Then, when calling `switch_to_locale( 'es_ES' );`, there is no cache entry for `es_ES` yet, so `WP_Textdomain_Registry::get_path_from_lang_dir()` looks up files in `WP_LANG_DIR`, not finding anything though
4. Same for the switch to `de_DE`
5. No translations applied -> test fails

Reply:
Fixed in 067a7fc649a0e80d5bbb30b9f0d3ca7896fe415d